src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.gtk.cs

```diff
@@ -146,19 +151,23 @@ void OnMouseMotion(object source, MotionNotifyEventArgs args)
 				using var ctx = new Context(surface);
 				DrawPoint(ctx, point);
 
-				area!.QueueDraw();
+				area?.QueueDraw();
 			}
 		}
 
 		void DrawPoint(Context ctx, PointD pointD)
 		{
-			ctx.SetSourceRGBA(currentLine!.LineColor.R, currentLine!.LineColor.G, currentLine!.LineColor.B, currentLine!.LineColor.A);
-			ctx.LineWidth = currentLine.LineWidth;
+			if (currentLine is not null)
+			{
+				ctx.SetSourceRGBA(currentLine.LineColor.R, currentLine.LineColor.G, currentLine.LineColor.B, currentLine.LineColor.A);
+				ctx.LineWidth = currentLine.LineWidth;
+			}
+
 			ctx.MoveTo(previousPoint);
 			previousPoint = pointD;
 			ctx.LineTo(pointD);
 			ctx.Stroke();
-			currentLine.Points.Add(new Point(pointD.X, pointD.Y));
+			currentLine?.Points.Add(new Point(pointD.X, pointD.Y));
```

Comment:
Could this go inside the `if (currentLine is not null)` and avoid the need for the `?`

Reply:
Fix me if I am wrong but it’s a short form of writing it. 
`
instance?.Method()
`

is the same as 
`
if (instance is not null)
    instance.Method()
`

Reply:
Yes it is the same, I actually meant inside the if higher up on line 160. Only a very minor point

Reply:
ahh got you, just move it a few lines up. yes. Let's do it!